setup.py

```diff
@@ -15,6 +15,28 @@
 LONG_DESCRIPTION_FILE = os.path.join(TOPDIR, 'README.rst')
 VERSION_FILE = os.path.join(TOPDIR, 'shoop', '_version.py')
 
+# Release instructions
```

Comment:
You only need these when releasing. It's just cruft for those who don't do releases.

On the other hand, when releasing, the first thing you try to accomplish, is to update the version number, which you'll find in setup.py. That's why it's the most visible place for instructions like this. I can move them to doc and add a "link" though, but I'm a bit scared that then nobody follows the "link".